_use_cases/continental.md

```diff
@@ -0,0 +1,22 @@
+---
+project: Keyless car sharing
+quote: RIOT is fun to hack. We value the modular design and the open-minded community.
```

Comment:
Vincent told me he'd be fine with this quote:
"We use RIOT in our after-market product for connected car-sharing. RIOT's modular architecture helped us a lot to address the multiple challenges we faced, to design software and networking embedded on a low-power 32-bit microcontroller. Also: RIOT is fun to tinker with, and the RIOT community is very open minded!"

and this logo: https://www.carlogos.org/tire-logos/continental-logo-black-on-gold.png

Can you modify?

Reply:
I see this has been merged faster than I could react... I'll PR changes.

Reply:
done.